src/acuvity/apexextend.py

```diff
@@ -270,12 +247,34 @@ def __build_police_request(
                     raise ValueError("annotations must be strings")
             request.annotations = annotations
 
-        # bypass_hash must be a string
-        if bypass_hash is not None:
-            if not isinstance(bypass_hash, str):
-                raise ValueError("bypass_hash must be a string")
-            request.bypass_hash = bypass_hash
+        # now here check the guard config and parse it for the analyzers, redaction and keywords.
+        if guard_config:
+            if guard_config.keywords and keywords:
```

Comment:
no, please just make these a general check: when you pass a guard_config, you cannot set keywords and redactions. They are totally alternate configuration mechanisms of your requests. If you don't use guard_config, then you have the freedom to change the request on your own, otherwise guard_config takes over - independently if you are using a guard that makes use of redactions or keywords or not.